tests/unit/checkpoint/test_zero_optimizer.py

```diff
@@ -161,8 +153,8 @@ def test_hybrid_optimizer_state(self, tmpdir, zero_stage):
 
     @pytest.mark.parametrize('zero_stage', [0, 1, 2, 3])
     def test_load_module_only(self, tmpdir, zero_stage):
-        if not get_accelerator().is_fp16_supported():
-            pytest.skip("fp16 is not supported")
+        if zero_stage == 0 and get_accelerator().device_name() == "cpu":
```

Comment:
The root cause needs more investigation.  It seems that the failure for zero 0 relates to allgather behavior.

The skip clauses needs to  be seperated into three groups:
1. the test is inherently fp16 or intend to test fp16 behavior -- skip if fp16 is not supported
2. the test fails because a known accelerator feature is missing (i.e. missing op builder) -- skip if that feature is missing
3. the test fails because of unknown/unclear reason -- skip if accelerator is cpu and plan for fix with future PR

So skip against CPU accelerator is a marker that further investigation is needed.

Reply:
@delock, you are correct, as always :). Can you please create an issue for separating the skip clauses, and we will handle it. Thanks!